docs/getting-started/install-kura.md

```diff
@@ -95,9 +95,8 @@ To have all the Kura features working, the following dependencies are required:
 - General: `setserial`, `zip`, `gzip`, `unzip`, `procps`, `usbutils`, `socat`, `gawk`, `sed`, `inetutils-telnet`.
 - Security: `polkit` or `policykit-1`, `ssh` or `openssh`, `openssl`, `busybox`, `openvpn`.
 - Bluetooth: `bluez` or `bluez5`, `bluez-hcidump` or `bluez5-noinst-tools`.
-- Time: `chrony`, `chronyc`, `cron` or `cronie`.
+- Time: `chrony`, `chronyc`
```

Comment:
Are the changes back ported a breaking change?

Reply:
Are you suggesting that, by removing the `cron` dependency from our package, we might break compatibility for a user that is currently leveraging `cron` in their bundle?

Reply:
yes